analyses/malawi/scripts/mwi_chirps_dry_spell_detection.R

```diff
@@ -35,15 +36,15 @@ mwi_adm2 <- st_read(paste0(shapefile_path, "/mwi_admbnda_adm2_nso_20181016.shp")
 # explore shapefiles
 summary(st_geometry_type(mwi_adm2)) # summary of geometry types
 st_crs(mwi_adm2) # coordinate reference system of shapefile
-st_bbox(mwi_adm2) # spatial extent 
-nrow(mwi_adm2) # number of features (= spatial objects / adm2 regions) 
+st_bbox(mwi_adm2) # spatial extent
+nrow(mwi_adm2) # number of features (= spatial objects / adm2 regions)
 ncol(mwi_adm2) # number of attributes
 names(mwi_adm2) # names of attributes
 mwi_adm2
 plot(mwi_adm2$geometry) # visual inspection
 
 mwi_adm2_spatial_extent <- st_bbox(mwi_adm2)
-mwi_adm2_ids <- as.data.frame(mwi_adm2) %>% dplyr::select('ADM2_PCODE', 'ADM2_EN') 
+mwi_adm2_ids <- as.data.frame(mwi_adm2) %>% dplyr::select('ADM2_PCODE', 'ADM2_EN')
 
```

Comment:
@joseepoirier this is probably due to that I opened the file and made small changes, but do you know how you can prevent R (studio) from adding/removing these spaces at the end of lines? Makes the PR very messy

Reply:
There is a formatting library but it has to be manually run to remove leading and trailing whitespaces. Adding to the Coding Practices" things to follow up on.